lib/Dialect/FIRRTL/Transforms/GrandCentralSignalMappings.cpp

```diff
@@ -435,11 +435,29 @@ void GrandCentralSignalMappingsPass::runOnOperation() {
   SmallVector<ModuleMappingResult> results;
   results.resize(modules.size());
 
+  auto updateStats = [this](auto &mappings) {
+    size_t sources = 0, sinks = 0;
+    for (auto &mapping : mappings) {
+      if (mapping.dir == MappingDirection::ProbeRemote)
+        ++sources;
+      else
+        ++sinks;
+    }
+    if (sources || sinks) {
```

Comment:
Any reason not to split this into two if blocks?

Reply:
Not a great reason, fixed!